Svc/PolyDb/PolyDb.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * PolyDbImpl.cpp
+ *
+ *  Created on: May 13, 2014
+ *      Author: Timothy Canham
+ */
+
+#include <Svc/PolyDb/PolyDb.hpp>
+#include <Fw/Types/Assert.hpp>
+#include <FpConfig.hpp>
+
+namespace Svc {
+    PolyDb::PolyDb(const char* name) : PolyDbComponentBase(name) {
+        // initialize all entries to stale
+        for (NATIVE_INT_TYPE entry = 0; entry < Svc::PolyDbCfg::PolyDbEntry::NUM_CONSTANTS; entry++) {
+            this->m_db[entry].status = MeasurementStatus::STALE;
+        }
+    }
+
+    void PolyDb::init(NATIVE_INT_TYPE instance) {
+        PolyDbComponentBase::init(instance);
```

Comment:
Delete the whole `init` function.  We have been removing them and using the parent `init`

Reply:
Removed.